HdmiCec_2/HDMICec_2.json

```diff
@@ -64,7 +123,7 @@
             }
         },
         "getOSDName":{
-            "summary": "Returns the OSD name used by device",
+            "summary": "Returns the OSD name used by device.\n  \n### Events \n\n No Events",
```

Comment:
name set by the user

Reply:
Updated changes as per your review comments